modules/users/client/controllers/password.client.controller.js

```diff
@@ -55,13 +53,13 @@
     function onRequestPasswordResetSuccess(response) {
       // Show user success message and clear form
       vm.credentials = null;
-      vm.success = response.message;
+      Notification.success({ message: response.message, title: 'Password request successful!' });
```

Comment:
On second thoughts, I actually get your point. The title is misleading. User is not requesting for a password. He is only requesting for a password reset email. I will add a couple of words and commit.